Jenkinsfile

```diff
@@ -0,0 +1,43 @@
+pipeline {
+  agent {
+    node {
+      label 'mac-os'
+    }
+    
+  }
+  stages {
+    stage('Build and test') {
+      when {
+        expression {
+          result = sh (script: "git log -1 | grep '.*\\[ci skip\\].*'", returnStatus: true) // Check if commit message contains skip ci label
+          result != 0 // Evaluate the result
+        }
+        
+      }
+      environment {
+        GIT = credentials('65325e52-5ec0-46a7-a937-f81f545f3c1b')
+      }
+      steps {
+        deleteDir()
+        checkout scm
+        dir(path: 'Yona') {
+          sh '/usr/local/bin/pod install'
+          sh 'xcrun agvtool new-marketing-version 1.1'
```

Comment:
What if we leave this command out? The current version is already 1.1 isn't it, so we don't have to update it with avgtool?

Reply:
I'd rather maintain the version number here, so we have a single place to look at, using the same approach for both Android and iOS